polaris-react/src/components/Banner/Banner.stories.tsx

```diff
@@ -142,6 +142,7 @@ export const InAModal = {
       <div style={{height: '500px'}}>
         <Button onClick={handleChange}>Open</Button>
         <Modal
+          instant
```

Comment:
Prevents the accessibility checker from running while the CSS fade-in transition for Modal is still happening and giving false positives for color contrast issues.